webui/react/src/fixtures/responses/experiment-details/non-scalar-metrics-4078.json

```diff
@@ -31,8 +31,8 @@
     "description": "Fork of Fork of mnist_tp_to_estimator_const",
```

Comment:
nit: we probably don't need to update anything under webui/react/src/fixtures.

Reply:
Yeah this is just part of the bumpenv.sh process. Unless it's worth adding this to an ignore-list in bumpenvs.sh, it's just automated to update this every time.